css/main.css

```diff
@@ -357,8 +358,25 @@ table th {
 		padding-bottom:0;
 	}
 	
-	#menu { 
-		display:none; 
+	#menu, 
+	#menu * { 
+		display:none!important;
+		visibility: hidden;
 	}
 	
+	.section .mod1, .section .mod1 h1, .mod1 { background-color:#000; color:#FFFFFF; }
```

Comment:
that doesn't do it, but this here does: ".section .col-md-2.col-xs-2, .section .col-md-2.col-xs-2 h1 { background-color: #000 !important; color: #FFFFFF !important; }
"

if this pr is included i will change the css